test/parallel/test-fs-buffertype-writesync.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+const common = require('../common');
+
+// This test insures that writeSync does support input buffers which
```

Comment:
^ I think this comment is still unaddressed? It’s just a comment but it’s really a bit confusing.